bynder_local.module

```diff
@@ -19,75 +24,94 @@ use GuzzleHttp\Exception\ConnectException;
 function bynder_local_media_presave(MediaInterface $media) {
   $source = $media->getSource();
```

Comment:
A quick summary of what you are doing here which is different to the default bynder_local functionality would be helpful.
It looks like you have created a config page where you can choose a Drupal field to save the Bynder asset to?

Reply:
I'll a few more lines under the summary description.

Reply:
Good